packages/core/src/new-api/internal/deployer.ts

```diff
@@ -26,30 +27,45 @@ import {
   DeploymentExecutionState,
   ExecutionStateMap,
 } from "./types/execution-state";
+import { TransactionLookupTimer } from "./types/transaction-timer";
 import { assertIgnitionInvariant } from "./utils/assertions";
 import { getFuturesFromModule } from "./utils/get-futures-from-module";
 import { validate } from "./validation/validate";
 
-const BLOCK_POLLING_INTERVAL = 200;
+// The interval between checks to see if a new block has been created
+const DEFAULT_BLOCK_POLLING_INTERVAL = 200;
+
+// In milliseconds the amount of time to wait on a transaction to
+// confirm before timing out
+const DEFAULT_TRANSACTION_TIMEOUT_INTERVAL = 3 * 60 * 1000;
```

Comment:
Switch from these being the hardcoded values to being the hard coded defaults. I went with 3 mins for default transaction timeout.